example/gluon/embedding_learning/get_cub200_data.sh

```diff
@@ -26,5 +26,8 @@ if [[ ! -d "${DATA_DIR}" ]]; then
   mkdir -p ${DATA_DIR}
 fi
 
+# the dataset is from Caltech-UCSD Birds 200
+# http://www.vision.caltech.edu/visipedia/CUB-200.html
+# for full text of the license, see https://creativecommons.org/licenses/by/4.0/legalcode
```

Comment:
Try to include all of the following information in both the README and comments: copyright, license, link to where they say what the license is.

```suggestion
# These datasets are copyright Caltech Computational Vision Group and licensed CC BY 4.0 Attribution.
# See http://www.vision.caltech.edu/archive.html for details
```